.travis.yml

```diff
@@ -80,4 +80,4 @@ jobs:
         - TRAVIS_JOB_NUMBER=WORKAROUND.1
       script:
         - npm run lint
-        - 'if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then npm run semantic-release; fi'
+        - 'if [ "${TRAVIS_PULL_REQUEST}" = "false" ] && [ "${TRAVIS_BRANCH}" = "master" ]; then npm run semantic-release; fi || true'
```

Comment:
Please remove it and rebase your branch instead

Reply:
Its okay now?
If not I can close this PR and new clean PR.